src/Hooks/HttpKernelResolvedHandler.php

```diff
@@ -27,19 +27,19 @@ public function __construct(
 
     public function __invoke(KernelContract $kernel, Application $app): void
     {
+        if (! $kernel instanceof Kernel) {
+            return;
+        }
+
```

Comment:
I'm moved this up. Might as well bail early.

It also aligns with what we do for the console kernel.